packages/odyssey-react-mui/src/themes/odyssey/typography.ts

```diff
@@ -21,52 +21,46 @@ export const typography: ThemeOptions["typography"] = {
   fontWeightRegular: Number(Tokens.FontWeightNormal),
   fontWeightMedium: 500,
   fontWeightBold: Number(Tokens.FontWeightBold),
-  h1: {
+  allVariants: {
```

Comment:
We can consolidate these styles. However, they don't seem to pass into our custom variants, so `body` still declares them.